pubspec.yaml

```
@@ -9,6 +9,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  flutter_hooks: ^0.18.0 
```

yamllint

[error] trailing spaces

(trailing-spaces)